mucommander-core/src/main/java/com/mucommander/ui/action/ActionKeymapWriter.java

```diff
@@ -35,92 +35,95 @@
 import com.mucommander.ui.text.KeyStrokeUtils;
 
 /**
- * This class is responsible for writing the actions.
- * When actions are modified, they are written to the user's actions file. 
- * 
+ * This class is responsible for writing the actions. When actions are modified, they are written to the user's actions
+ * file.
+ *
  * @author Maxence Bernard, Arik Hadas
  */
 class ActionKeymapWriter extends ActionKeymapIO {
-	private static final Logger LOGGER = LoggerFactory.getLogger(ActionKeymapWriter.class);
-	
-	ActionKeymapWriter() {}
-	
-	public void create() throws IOException {
-		BackupOutputStream bos = new BackupOutputStream(getActionsFile());
-
-		try {
-			new Writer(bos).writeKeyMap(null);
-		} catch (Exception e) {
-			LOGGER.debug("Caught exception", e);
-		} finally {
-			bos.close();
-		}
-	}
-	
-	void write() throws IOException {
-		Map<String, KeyStroke[]> combinedMapping = new Hashtable<String, KeyStroke[]>();
-		Iterator<String> modifiedActionsIterator = ActionKeymap.getCustomizedActions();
-
-		while(modifiedActionsIterator.hasNext()) {
-			String actionId = modifiedActionsIterator.next();
-			KeyStroke[] keyStrokes = new KeyStroke[2];
-			keyStrokes[0] = ActionKeymap.getAccelerator(actionId);
-			keyStrokes[1] = ActionKeymap.getAlternateAccelerator(actionId);
-
-			combinedMapping.put(actionId, keyStrokes);
-		}
-		
-		BackupOutputStream bos = new BackupOutputStream(getActionsFile());
-
-		try {
-			new Writer(bos).writeKeyMap(combinedMapping);
-			wereActionsModified = false;
-		} catch (Exception e) {
-			LOGGER.debug("Caught exception", e);
-		} finally {
-			bos.close();
-		}
-	}
-	
+    private static final Logger LOGGER = LoggerFactory.getLogger(ActionKeymapWriter.class);
+    private static final String ACTION_COMMENT_HEADER =
+            "See http://trac.mucommander.com/wiki/ActionKeyMap for information on how to customize this file";

```

Comment:
the link is broken (trac.mucommander.com is no longer used) and I don't see it in the wiki on github so it's better to rather drop this comment